services/simplified_crontab

```diff
@@ -77,7 +77,7 @@ HOME=/var/www/circulation
 
 # Overdrive
 #
-0 */4 * * * root core/bin/run overdrive_monitor_full >> /var/log/cron.log 2>&1
+0 3 * * * root core/bin/run overdrive_new_titles >> /var/log/cron.log 2>&1
```

Comment:
So this will run once a day at 3am, right?

Reply:
Yep.